src/main/webapp/styles/techstyle.css

```diff
@@ -2,20 +2,30 @@
 #left-tutorials-button {
   flex: 1;
   width: 50%;
-  border: solid 1px #000;
+}
+
+#left-tutorials-button li {
```

Comment:
As I mentioned during the meeting, a quick but significant improvement would be that all buttons on all pages have the same size and style, so you would move this into general.css and use a more generic class name so that all pages can use it.

Reply:
Added a consistent button to index.html for all pages to use.

Reply:
I don't see that in index.html: https://github.com/googleinterns/granite2020/pull/15/files/3e52934aefcb3d851bd159bd989b48c640734e30..fd2f5ec59437a4ca0b691422416201361757f21e#diff-747e7a37d03935942014106202566830

And wouldn't it be a class in general.css that's set on all buttons?

Reply:
Oops I see that I did that on a different branch. Most buttons are consistent based on the one appearing in the general.css : input[type=submit] . However on index.html Praneet pointed out that in order for the API to work correctly the login and signup needed to be links versus a form.

Reply:
general.css has general button styling but the styling for the buttons on index.html are in homestyle.css